conseil-common/src/main/scala/tech/cryptonomic/conseil/common/bitcoin/rpc/json/Transaction.scala

```diff
@@ -3,9 +3,7 @@ package tech.cryptonomic.conseil.common.bitcoin.rpc.json
 /**
   * Trait for transaction inputs and outputs.
   */
-trait TransactionComponent {
```

Comment:
Why not sealing it? It could help supporting exhaustive matching and provide some guarantee?